app/provider/Registry/ServiceProvider.php

```diff
@@ -53,7 +63,7 @@ public function register()
         $path = $this->path;
 
         $this->di->setShared(
-            $this->serviceName,
+            'registry',
```

Comment:
why?

Reply:
because maybe I think we should get rid of this variable and should use names instead variable.